src/installer/__tests__/index.ts

```diff
@@ -318,6 +318,37 @@ describe('install', (): void => {
     expect(exists('.git/hooks/pre-commit')).toBeFalsy()
   })
 
+  it('should not install hooks if searchParentDirs is false & cwd is git-less', (): void => {
```

Comment:
Typically I'd use a `describe` to group these tests, but didn't know what style you prefer (given lack of `describe` & use of `should`).